content/working_with/console/widgets/serviceButton.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: bt_wiki
+title: Service button
+category: Widgets
+draft: false
+---
+
+Displays a button allowing to upload a blueprint from the configured blueprint catalog/marketplace. 
+
+For more details about blueprint catalog/marketplace, check [Blueprint Catalog]({{< relref 
+"working_with/console/widgets/blueprintCatalog.md" >}}) widget.
+
+![service-button]( /images/ui/widgets/service-button.png )
+
+
+## Settings
+
+* `Color` - The color of the button. Available icons list can be found
+  at: [Theming - Semantic UI React](https://react.semantic-ui.com/layouts/theming). Default: ''
+* `Label` - The label displayed in the button. Default: 'Create a service'
+* `Icon` - Name of the icon displayed in the button. Available icons list can be found
+  at: [Icon - Semantic UI React](https://react.semantic-ui.com/elements/icon). Default: 'add'
+* `Marketplace tabs` - List of tab name and marketplace URL pairs. For the expected format of the data provided in 
+  the URL, see `Blueprint Examples URL` configuration field in
```

Comment:
I think 'the expected format of the data provided in the URL' is misleading, IIUC it refers to the structure of the JSON file located at the URL, not the data that is part of the URL itself.

Reply:
I'll remove 'the expected format' part right now. I think I'll need to wait for Blueprint Catalog and Blueprints widget update first.